app/views/signatures/_form.html.erb

```diff
@@ -14,7 +20,11 @@
   <%= f.text_field :name, :tabindex => increment, :maxlength => 255, :size => nil, :class => 'form-control'  %>
 <% end %>
 
-<%= render 'signatures/email', f: f, signature: signature %>
+<%= form_row :for => [signature, :email] do %>
+  <%= f.label :email, 'Email address', :class => 'form-label' %>
+  <%= error_messages_for_field signature, :email %>
+  <%= f.text_field :email, :type => 'email', :tabindex => increment, :class => 'form-control' %>
+<% end %>
```

Comment:
The only difference between this form_row and the one in the new ``signatures/replay-email`` partial seems to be that the label is hidden in the partial, but not here.  Could we use a parameter to the partial to control that and go back to using the same partial in all places?